### ISSUE TITLE
Create teaser component in Storybook

### ISSUE THREAD

**Issue by SvSven** [equinor/energyvision#310]

**Comment by wenche** [comment]
We still need to discuss a couple of things:

- Which CSS variables to use for spacings and font sizes
- Mobile design
- One column vs two column
- The 800px maximum height
- The ingress styles

**Comment by wenche** [comment]
@BirteThornquist  and I will do the last walkthrough of this tomorrow. Friday. We probably still need to adjust some spacings slightly.

**Comment by wenche** [comment]
Talked about this with Birte and Sven. We decided to create a new, separate issue for inline SVG that will give us more flexibility with responsive SVG. But this was not supported out of the box by Sanity. Now it's respecting the original aspect ratio of the SVG and center the image if it's too small for the content.

![Skjermbilde 2021-05-21 kl. 14.14.12.png](https://images.zenhubusercontent.com/5fb7aa9b653d5135bb8c50f5/cb507139-25c6-4522-a4c1-d715c301a2ff)

We added some more spacing and decided to remove the maximum height of 800px to avoid mistakes. But we need to clarify content length as good as possible in the Sanity studio. @SvSven  will fix the warning for the text input to actually work. :thumbsup:

[ Link to test page with some teasers](https://energyvision.app.radix.equinor.com/en/careers)
